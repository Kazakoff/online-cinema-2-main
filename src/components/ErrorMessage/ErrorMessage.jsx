import React from 'react';

export const ErrorMessage = (props) => {
    return (
        props.error ?
            <p style={{ color: 'red', fontSize: '12px', fontWeight: '400' }} >{props.error}</p>
            :
            null
    )
}