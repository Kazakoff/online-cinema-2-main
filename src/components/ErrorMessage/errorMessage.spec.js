import React from 'react'
import { ErrorMessage } from './ErrorMessage'

const props = {
    error: 'some error'
};

const setUp = (props) => shallow(<ErrorMessage {...props} />);

describe('ErrorMessage component', () => {
    it('should render ErrorMessage component', () => {
        const component = setUp(props)
        expect(component.debug()).toMatchSnapshot()
    })
})