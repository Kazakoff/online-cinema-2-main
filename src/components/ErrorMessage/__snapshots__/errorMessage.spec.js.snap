// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage component should render ErrorMessage component 1`] = `
"<p style={{...}}>
  some error
</p>"
`;
